### PR TITLE
Enable the Council to delegate ruling of specific FOs

### PR DESCRIPTION
**Pull request by frivoal**
See #618

***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/621.html" title="Last updated on Aug 24, 2022, 4:24 PM UTC (32a6f0c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/621/af2bad3...frivoal:32a6f0c.html" title="Last updated on Aug 24, 2022, 4:24 PM UTC (32a6f0c)">Diff</a>

### CODE REVIEW

**Comment by frivoal** [comment]
This implements the following AB resolution:
> RESOLUTION: The Council may decide by super-majority to defer a decision to a suitable officer, employee, or the Board.

File: index.bs [hunk 1]
```diff
@@ -2299,7 +2299,17 @@ as well as <a href="https://docs.google.com/document/d/1lB8Q_pvqxfN204FQHZ8D2HOb
 	falling back to a vote if that fails.
 	The chair must be a member of the [=W3C Council=].
 
-	Having reviewed the information gathered by the [=Team=] and its recommendation,
+	In extraordinary cases,
+	if the [=Council=] determines that it would not be the appropriate body to handle a particular [=Formal Objection=],
+	it <em class=rfc2119>may</em> decide by a ⅔ supermajority vote
```
Comment (TallTed) [inline index.bs:2304]:
```suggestion
	it <em class=rfc2119>may</em> decide by a two-thirds supermajority vote
```

File: index.bs [hunk 2]
```diff
@@ -2299,7 +2299,17 @@ as well as <a href="https://docs.google.com/document/d/1lB8Q_pvqxfN204FQHZ8D2HOb
...
+	(i.e. twice as many votes in favor as against)
+	to defer the decision for that specific [=Formal Objection=]
```
Comment (TallTed) [inline index.bs:2306]:
```suggestion
	to delegate the decision for that specific [=Formal Objection=]
```
or possible `refer`, but definitely not `defer`

**Comment by frivoal** [comment]
> I think we should go back to the supermajority question

No strong opinion from me on that at the moment, but if so, we should circle back to the AB, as their resolution explicitly included supermajority.

**Pull request merged**